CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 This project uses [semantic versioning](http://semver.org/).
 
+## [2.3.3] 2022-10-31
+Pick up the latest marked-it-core.
```

Comment:
Make mention of the new syntax support and the addition of the command line switch that can disable it.